### PR TITLE
feat: add Accent Box Container Component #698

### PR DESCRIPTION
**Pull request by juandiegocv27**
create the accent box, easy and optimal to be implemented in any part of the web:

WhatsApp Image 2024-05-30 at 18 43 34_1fbddb72
WhatsApp Image 2024-05-30 at 18 43 42_d59f42a9

## Summary by CodeRabbit

- **New Features**
    - Introduced a new `AccentBox` component for better content display and expandability.

- **Enhancements**
    - Wrapped various components with the `AccentBox` for a consistent and customizable UI experience. Components affected include:
        - Step Element
        - Timer
        - Analytics Skeleton
        - Stat Element

### CODE REVIEW

**Comment by juandiegocv27** [comment]
@ayushtom already created the PR and fix some errors, waiting for feedback🫡.

**Comment by ayushtom** [comment]
Screenshot 2024-05-31 at 10 56 55 PM



Screenshot 2024-05-31 at 10 57 08 PM


hey i think im not sure why the new changes have been applied to the outside container here. They dont look the same

**Comment by juandiegocv27** [comment]
@ayushtom create code as needed:
image

File: components/UI/AccentBox.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
```
[the remark this answers is not in the document]
Comment (juandiegocv27) [reply components/UI/AccentBox.tsx]: I think I will let it be like this:
image

File: components/UI/AccentBox.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    }
+  };
+
+  return (
+    <div
+      className={`${styles.accentBox} ${className} ${isExpanded ? styles.expanded : ''}`}
+      style={{ ...style }}
+      onClick={handleToggle}
+      onKeyUp={handleKeyUp}
+      tabIndex={0}
```
[the remark this answers is not in the document]
Comment (juandiegocv27) [reply components/UI/AccentBox.tsx]: ok got it, but I'll let it like this

[the remark this answers is not in the document]
Comment (juandiegocv27) [reply components/UI/AccentBox.tsx]: image

File: components/UI/AccentBox.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  return (
+    <div
+      className={`${styles.accentBox} ${className} ${isExpanded ? styles.expanded : ''}`}
+      style={{ ...style }}
+      onClick={handleToggle}
+      onKeyUp={handleKeyUp}
+      tabIndex={0} 
+    >
+      <div className={styles.gradientLine}></div>
```
[the remark this answers is not in the document]
Comment (juandiegocv27) [reply components/UI/AccentBox.tsx:20]: its just a Div usage, it stil works for me like this: 
image
and it helps visually for the next lines

**Comment by ayushtom** [comment]
hey

i think if you apply the component to the highlighted box below (for both the boxes) then that would be perfect. Also a little thinner linear gradient would look nice too

Screenshot 2024-06-03 at 11 23 45 AM

**Comment by juandiegocv27** [comment]
@ayushtom 
WhatsApp Image 2024-06-03 at 16 29 39_cf745603

**Comment by juandiegocv27** [comment]
@ayushtom waiting for feedback

**Comment by juandiegocv27** [comment]
image
@ayushtom  accentbox integrated to more part of the website

**Comment by juandiegocv27** [comment]
@ayushtom already put accent box in every part of the web as required, thank you for the feedback

**Comment by juandiegocv27** [comment]
@ayushtom ready to merge 🏆

**Review by ayushtom — APPROVED** [review]
lgtm

**Pull request merged**